src/main/java/com/minecolonies/core/colony/managers/CitizenManager.java

```diff
@@ -224,15 +225,15 @@ public void sendPackets(
     @Override
     public ICitizenData spawnOrCreateCivilian(@Nullable final ICivilianData data, final Level world, final BlockPos spawnPos, final boolean force)
     {
-        if (!colony.getBuildingManager().hasTownHall() || (!colony.canMoveIn() && !force))
+        if (!colony.getServerBuildingManager().hasTownHall() || (!colony.getSettings().getSetting(BuildingTownHall.MOVE_IN).getValue() && !force))
```

Comment:
Eh why was this changed, these getters still exist on the colony no? They were modified to do this call internally, so why does this need changing?